Gem/gem.json

```diff
@@ -1,10 +1,21 @@
 {
-    "GemFormatVersion": 4,
-    "Uuid": "5111dee500ed4812a7905093a73d3837",
-    "Name": "NetSoakTest",
-    "DisplayName": "NetSoakTest",
-    "Summary": "NetSoakTest Project Gem. Indefinitely tests AzNetworking transport via loopback.",
-    "Tags": ["Game"],
-    "IconPath": "preview.png",
-    "IsGameGem": true
+    "gem_name": "NetSoakTest",
+    "display_name": "NetSoakTest",
+    "licence": "Apache-2.0 or MIT",
+    "license_url": "https://github.com/o3de/o3de-netsoaktest/blob/development/LICENSE.txt",
+    "origin": "",
+    "origin_url": "https://github.com/o3de/o3de-netsoaktest",
+    "type": "",
+    "summary": "NetSoakTest Project Gem. Indefinitely tests AzNetworking transport via loopback.",
```

Comment:
nit:  Will defer to @AMZN-puvvadar for concise description.

Reply:
As presently constructed, this is accurate.